apps/web/drizzle.config.ts

```diff
@@ -4,9 +4,10 @@ import env from './src/env'
 
 const testEnv = process.env.NODE_ENV === 'test'
 const connectionString = testEnv ? env.TEST_DATABASE_URL : env.DATABASE_URL
+
 export default {
   dialect: 'postgresql',
-  schema: './src/db/schema/index.ts',
+  schema: './node_modules/@latitude-data/database/src/schema/index.ts',
```

Comment:
The drizzle.config.ts still lives in NextJS because is where the `db:migrate` command and the rest are executed. The difference is that we pick schema from the database package. Not sure if this will be a problem in some way

Reply:
spoiler: it is 😅 

Since the database package has its own typescript config doesn't know about, it can't do stuff like resolve custom path aliases in the database package. I'm trying to fix this atm.